apps/supernova/src/App.jsx

```diff
@@ -62,13 +59,31 @@ function App(props = {}) {
   )
 }
 
+const AppWithOldUrlStructure = (props) => {
+  // syncs navigation relevant states with the url for deep links
+  // gets the state from the URL in the beginning
+  // sets the URL from state information
+  useUrlState()
```

Comment:
This was somehow reintroduced by a refactoring of @TilmanHaupt as I can see the history. Thats why we used to have a component returning null to encapsulate this re-renders. We might need to thing a better solution instead of spreading the sync of the url over the code...

Reply:
Yeah, this name is missleading as we spoke about it...

Reply:
Please create a ticket :)